src/razor/src/Completion/RazorCompletionItemProvider.ts

```diff
@@ -71,6 +66,13 @@ export class RazorCompletionItemProvider
                 completions instanceof Array ? completions  // was vscode.CompletionItem[]
                     : completions ? completions.items       // was vscode.CompletionList
                         : [];
+            
+            // For CSharp, completions need to keep the "data" field
```

Comment:
Comment moved down - I feel like it makes more sense here.

Reply:
Thank you :)

Reply:
Oh actually, this is subtle... it might make sense to put before the call to `RoslynLanguageServer.provideCompletionsCommand`. It's the reason that command exists